beacon-chain/chaintest/backend/setup_db.go

```diff
@@ -25,7 +27,7 @@ func teardownDB(db *db.BeaconDB) {
 	if err := db.Close(); err != nil {
 		log.Fatalf("failed to close database: %v", err)
 	}
-	path := path.Join(os.TempDir(), "/simulateddb")
+	path := path.Join(os.TempDir(), fmt.Sprintf("/%d", rand.Int()))
```

Comment:
This won't clean up the same random number will it?

I think you need to keep track of this number somehow, maybe this path could be `db.DatabasePath`

Reply:
+1

Reply:
Yeah agreed, maybe try to keep track of the number of simulated backend instances running and assign your path that way.